### PR TITLE
[DEVX-662]: Addition of Periodic Testing on Staging & Prod

### PR DESCRIPTION
**Pull request by srikanthbachala20**
### Why

* To resolve issues in SDK as soon as it appeared

### How

* Run SDK tests periodically and alert on failure of these tests

### CODE REVIEW

File: .github/workflows/run_tests_prod.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Run Tests - Prod
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6,18 * * *'
```
Comment (nitinbhojwani) [inline .github/workflows/run_tests_prod.yml:6]: Sets up a cron job to automatically run the workflow twice a day, at 6:00 AM and 6:00 PM (UTC)

File: .github/workflows/run_tests_prod.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  sdk-python-tests-prod:
+    uses: Clarifai/clarifai-python/.github/workflows/run_tests.yml@DEVX-662-Periodic-Synthetic-Checks
```
Comment (mogith-pn) [inline .github/workflows/run_tests_prod.yml:10]: Should we need to change this to Master branch once it is merged ?

Comment (srikanthbachala20) [reply .github/workflows/run_tests_prod.yml:10]: Ohh Yeah, i forgot to change this back. This helps me to test the workflows using current branch. You can leave this comment unresolved. When the PR is Approved, I will resolve this accordingly.

File: .github/workflows/run_tests_staging.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: Run Tests - Staging
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6,18 * * *'
+
+jobs:
+  sdk-python-tests-staging:
+    uses: Clarifai/clarifai-python/.github/workflows/run_tests.yml@DEVX-662-Periodic-Synthetic-Checks
```
Comment (sanjaychelliah) [inline .github/workflows/run_tests_staging.yml:10]: Mogith's comment applies here too. Leaving this unresolved for now!

**Review by sanjaychelliah — APPROVED** [review]

**Pull request merged**